### ISSUE TITLE
Unconditional accessibility warnings

### ISSUE THREAD

**Issue by gagan0723** [evcohen/eslint-plugin-jsx-a11y#320]
### Error:
* 'jsx-a11y/img-has-alt' 
* 'jsx-a11y/href-no-hash' was not found

![log](https://user-images.githubusercontent.com/6944095/30122660-83abba56-932f-11e7-9806-00fb595dc3ba.png)
### index.js content
![index](https://user-images.githubusercontent.com/6944095/30122664-8914e7a6-932f-11e7-94c6-c896c75ce159.png)
### package.json
![package](https://user-images.githubusercontent.com/6944095/30122667-8d20a8f8-932f-11e7-9c0f-de1263c20c27.png)
### .eslintrc extends
![extends](https://user-images.githubusercontent.com/6944095/30123068-07dd518a-9331-11e7-8513-e48add3a17c8.png)
### .eslintrc plugins
![lint](https://user-images.githubusercontent.com/6944095/30122668-8f113b8c-932f-11e7-8876-4a2e6e1e548d.png)
### .eslintrc rules
![a](https://user-images.githubusercontent.com/6944095/30122718-c4bc448e-932f-11e7-8767-0a75fd0e321a.png)

**Comment by ljharb** [comment]
What does `npm ls > /dev/null` do? If it exits nonzero, nothing can be expected to work.

**Comment by gagan0723** [comment]
I get two errors.
```
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^5.1.1, required by eslint-config-airbnb@15.1.0
npm ERR! extraneous: ajv@5.2.2 /home/user/Development/Web/component-lib/node_modules/ajv-keywords/node_modules/ajv
```
Is **ajv** a **JSON-Schema Validator?**

**Comment by evcohen** [comment]
You should update this plugin to ^v5.1.1 if you're using `eslint-config-airbnb.` Instructions to install this plugin from airbnb's config is [here](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/README.md#installation)

**Comment by gagan0723** [comment]
@evcohen  Updated eslint-plugin-jsx-a11y to ^v5.1.1. But linter still logs same error.

**Comment by ljharb** [comment]
You will probably need to restart your editor after any node_modules changes.

**Comment by gagan0723** [comment]
Great it works.
